setup.cfg

```diff
@@ -14,6 +14,9 @@ classifiers =
   Operating System :: POSIX :: Linux
   License :: OSI Approved :: MIT License
   Programming Language :: Python :: 3
+  Programming Language :: Python :: 3.12
+  Programming Language :: Python :: 3.11
+  Programming Language :: Python :: 3.10
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.